helm/rtd/values-dev.yaml

```diff
@@ -20,25 +20,28 @@ microservice-chart:
       memory: "8Gi"
       cpu: "1"
 
-  #  autoscaling:
-  #    enable: false
-  #    minReplica: 1
-  #    maxReplica: 2
-  #    pollingInterval: 30 # seconds
-  #    cooldownPeriod: 300 # seconds
-  #    triggers:
-  #      - type: azure-monitor
-  #        metadata:
-  #          tenantId: 7788edaf-0346-4068-9d79-c868aed15b3d
-  #          subscriptionId: ac17914c-79bf-48fa-831e-1359ef74c1d5
-  #          resourceGroupName: dvopla-d-sec-rg
-  #          resourceURI: Microsoft.KeyVault/vaults/dvopla-d-neu-kv
-  #          metricName: ServiceApiHit
-  #          # metricNamespace: Microsoft.KeyVault/vaults
-  #          # metricFilter: namespace eq 'default'
-  #          # metricAggregationInterval: "-1:1:0"
-  #          metricAggregationType: Count
-  #          targetValue: "30"
+topologySpreadConstraints:
```

Comment:
can you check the indentation? this property is at root level unlike the other values files modified in this PR